docs(reference/contenttypeparser): make more concise (#5950)
fastify/fastify: docs/Reference/ContentTypeParser.md

```

Fastify first tries to match a content-type parser with a `string` value before
trying to find a matching `RegExp`. For overlapping content types, it starts

> with the last one passed and ends with the first. To specify a general content

Reviewer: ```suggestion
with the last one configured and ends with the first (last in, first out). To specify a general content
```

```

#### removeAllContentTypeParsers
The `removeAllContentTypeParsers` API removes all existing content type parsers

> eliminating the need to specify each one individually. This API support encapsulation

Reviewer: ```suggestion
eliminating the need to specify each one individually. This API supports encapsulation
```

})
```

> **Notice**: The old syntaxes `function(req, done)` and `async function(req)` are

Reviewer: ```suggestion
**Notice**: `function(req, done)` and `async function(req)` are
```
